src/main/java/Task.java

```diff
@@ -14,15 +14,15 @@ public void markAsDone() {
         this.isDone = true;
         System.out.println(lineBreaker);
         System.out.println("Nice! I've marked this task as DONE:");
-        System.out.println(this.toString());
+        System.out.println("  " + this.toString());
         System.out.println(lineBreaker);
     }
 
     public void markAsUndone() {
```

Comment:
Perhaps you can consider changing the method name to setUndone() since "Setter methods for boolean variables must be of the form: void setFound(boolean isFound);"